torch/csrc/distributed/c10d/comm.h

```diff
@@ -33,14 +41,30 @@ class GradBucket {
     return tensors_;
   }
 
+  const std::vector<size_t>& getOffsets() const {
+    return offsets_;
```

Comment:
what are offsets in this context?

Reply:
Start index of the variable in the flattened 1D tensor. Added a comment to describe this API.